### ISSUE TITLE
PHP7 support for Debian.

### ISSUE THREAD

**Issue by Applepiee** [Eugeny/ajenti-v#225]
Hi!

I noticed that it was impossible for me to upgrade to PHP7 Ajenti-V on my Debian server.

When will you guys start supporting debian?

Regards,
Davy

**Comment by Xefir** [comment]
php7 isn't on stable repository, but you can install yourself if you are on stretch.

It's not ajenti's fault, debian tends to let things goes and integrate it only on major release.

Oh and btw, the php package has been rename for php7 to be able to install both php5 and php7 on the same machine.

Please see this PR witch talk about it : https://github.com/Eugeny/ajenti-v/pull/213

**Comment by armpdq** [comment]
Xefir, #213 is about ubuntu, not debian

**Comment by Xefir** [comment]
Yes, it's why i wrote "but you can install yourself if you are on stretch."

If the same package name as ubuntu : https://packages.debian.org/stretch/php7.0

You simply don't have to run the apt-add-repository command since it was on debian's repository

**Comment by Ardakilic** [comment]
You can follow this link for Ondřej Surý's repository on Debian Jessie and Stretch:

https://packages.sury.org/php/README.txt

I've ben using this for a while, and it never let me down so far both on Ubuntu and Debian.
